src/OpenTK.Windowing.Common/Utils.cs

```diff
@@ -0,0 +1,15 @@
+using System.Runtime.CompilerServices;
+
+namespace OpenTK.Windowing.Common
+{
+	public static class Utils
```

Comment:
Should this be in this namespace or OpenTK.Core?
Is there really not a standard library function for this?

Reply:
I'm referring to `Swap<T>(ref T, ref T)`

Reply:
There was in VB, I don't know if there still is. There is nothing in .net itself or in C#.